app/src/main/java/org/ole/planet/myplanet/ui/feedback/AdapterFeedback.kt

```diff
@@ -30,13 +30,13 @@ class AdapterFeedback(private val context: Context, private var list: List<Realm
                 "${context.getString(R.string.open_date)}: ${getFormatedDate(list?.get(position)?.openTime)}"
         rowFeedbackBinding.feedbackCardView.contentDescription = contentDescription
 
-        if ("yes".equals(list?.get(position)?.priority, ignoreCase = true)) {
+        if (context.getString(R.string.yes).equals(list?.get(position)?.priority, ignoreCase = true)) {
```

Comment:
@Kuljeet1998 this can't be translated at the moment as we compare it with data from the couch db which will always be in english. This change will make the condition to always fail